lsst/ts/cbp/statemachine.py

```diff
@@ -231,159 +237,28 @@ def __init__(self, port: str, address: int, frequency: float = 2, initial_state:
-            The value of the azimuth to move to in degrees.
-
-        Returns
-        -------
-        None
-
-        """
-        move_azimuth_topic = self.remote.cmd_moveAzimuth.DataType()
-        move_azimuth_topic.azimuth = azimuth
-        move_azimuth_ack = await self.remote.cmd_moveAzimuth.start(move_azimuth_topic,timeout=timeout)
-        self.log.info(move_azimuth_ack.ack.ack)
-
-    async def move_altitude(self, altitude, timeout=10):
-        """Calls the moveAltitude command and awaits and logs the acknowledgement code.
-
-        Parameters
-        ----------
-        altitude: int
-            The altitude in degrees to move CBP to.
+    def configure(self, config):
+        try:
+            self.model.configure(config)
+        except Exception as e:
+            self.log.error(e)
 
-        Returns
-        -------
-        None
-
-        """
-        move_altitude_topic = self.remote.cmd_moveAltitude.DataType()
-        move_altitude_topic.altitude = altitude
-        move_altitude_ack = await self.remote.cmd_moveAltitude.start(move_altitude_topic,timeout=timeout)
-        self.log.info(move_altitude_ack.ack.ack)
-
-    async def set_focus(self,focus, timeout=10):
-        """Sends the setFocus command and awaits and logs the acknowledgement code.
-
-        Parameters
-        ----------
-        focus: int
-            The focus in microns to set the focus encoder to.
-
-        Returns
-        -------
-        None
-
-        """
-        set_focus_topic = self.remote.cmd_setFocus.DataType()
-        set_focus_topic.focus = focus
-        set_focus_ack = await self.remote.cmd_setFocus.start(set_focus_topic,timeout=timeout)
-        self.log.info(set_focus_ack.ack.ack)
-
-    async def change_mask(self, mask, timeout=10):
-        """Calls the changeMask command and awaits and logs the acknowledgement code.
-
-        Parameters
-        ----------
-        mask: str
-            The string of the mask name to change to.
-        Returns
-        -------
-        None
-
-        """
-        change_mask_topic = self.remote.cmd_changeMask.DataType()
-        change_mask_topic.mask = mask
-        change_mask_ack = await self.remote.cmd_changeMask.start(change_mask_topic, timeout=timeout)
-        self.log.info(change_mask_ack.ack.ack)
-
-    async def park(self,timeout=10):
-        """Calls the park command and awaits and logs the acknowledgement code.
-
-        Returns
-        -------
-        None
-
-        """
-        park_topic = self.remote.cmd_park.DataType()
-        park_ack = await self.remote.cmd_park.start(park_topic,timeout=timeout)
-        self.log.info(park_ack.ack.ack)
+    def get_config_pkg(self):
+        return "ts_config_mtcalsys"
 
+    async def implement_simulation_mode(self,simulation_mode):
+        if simulation_mode == 0:
+            self.model._cbp.set_simulation_mode(simulation_mode)
+        elif simulation_mode == 1:
+            self.model._cbp.set_simulation_mode(simulation_mode)
```

Comment:
I suggest adding ; a final `else` that raises a `salobj.ExpectedError` for any other value. Otherwise the code will silently ignore invalid values, which could quite confusing.